itou/templates/apply/includes/job_applications_filters/top_filters.html

```diff
@@ -1,6 +1,6 @@
 {% load django_bootstrap5 %}
 
-<div class="s-section__row row">
+<div class="s-section__row row selection-indicator" data-emplois-elements-visibility-on-selection="hidden">
```

Comment:
Pourquoi pas `data-it-* (it pour itou comme l'app django) au lieu de `data-emplois-*` ?

Reply:
L'idée serait que `data-bs-*` c'est bootstrap, `data-it-*` c'est le thème et `data-emplois-*` serait du spécifique projet.

Reply:
Why not

Reply:
> L'idée serait que data-bs-* c'est bootstrap, data-it-* c'est le thème et data-emplois-* serait du spécifique projet.

Excellente idée ! Je ne sais pas s'il y a déjà des data-* spécifiques aux emplois mais, si c'est le cas, ça mériterait une entrée dans le migrations.md et un petit message pendant un point tech.